### PR TITLE
Swagger input validation

### PR DESCRIPTION
**Pull request by Meketh**
### What does this Pull Request do?
Adds input validation using swagger-tools.

### Are there points in the code the reviewer needs to double check?
Swagger docs now execute validation code.

### Does this Pull Request meet the acceptance criteria?
- [x] I have read the [Contribution guidelines](https://github.com/wibsonorg/notary-sdk/CONTRIBUTING.md).
- [x] I have read the [Code of Conduct](https://github.com/wibsonorg/notary-sdk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Agus** [comment]
Should we remove the already deprecated endpoints?

File: notary-api/src/routes/buyers.js [hunk 1]
```diff
@@ -212,23 +210,28 @@ router.post(
...
+ *             - callbackUrl
+ *           properties:
+ *             orderId:
+ *               type: integer
+ *               description: The unique identifier for the order
+ *             sellers:
+ *               type: array
+ *               description: List of sellers send to notarize
+ *               items:
+ *                 type: string
```
Comment (nicoayala) [inline notary-api/src/routes/buyers.js]: This is also expected as an object
```suggestion
 *                 type: object
```

**Comment by nicoayala** [comment]
I agree with @Agus that instead of deprecating we could delete those useless routes, but I'd suggest doing a cleanup when we finish updating to v2, there are a lot of things to remove from earlier versions. Let's focus on the priorities for now.

**Comment by nicoayala** [comment]
`eslint` is breaking the build with many `'<dependency>' should be listed in the project's dependencies...` errors, which is weird because all those deps are correctly declared 🤷‍♂️ , happens locally also

**Comment by Meketh** [comment]
Yeah, that's weird.
I'm looking into it but I can't seem to find a solution.

**Comment by nicoayala** [comment]
I'd try updating `eslint` and if that doesn't do the trick, I'd `eslintignore`d it

File: notary-api/src/utils/wibson-lib/cryptography/encription.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (Agus) [inline notary-api/src/utils/wibson-lib/cryptography/encription.js]: Remove all this `eslint-disable` lines. You don't have to ignore them as long as we delete this intruder: https://github.com/wibsonorg/notary-sdk/blob/ns-swagger-input-validation/notary-api/src/utils/wibson-lib/package.json 🙂 ... After that, lint will work.

**Pull request merged**